db/migrate/20241116011452_add_footer_to_feeds.rb

```diff
@@ -0,0 +1,5 @@
+class AddFooterToFeeds < ActiveRecord::Migration[7.2]
+  def change
+    add_column :feeds, :footer, :string
```

Comment:
`description_footer` or `episode_footer` might be a bit more precise?  We do have some `episode_***` fields on the feeds already, for stuff that applies at the ep-level rather than podcast/feed.

But I'm fine with this too.

Reply:
it only makes sense to add a footer to episodes (if it was in the podcast/feed, it would just be at the bottom of the single description)

Reply:
I could rename `episode_footer`, I thought about that as an option too

Reply:
I'm pro `episode_footer` - just something to indicate it's not the footor-for-this-feed's-description or something.  Since feeds can inherit a description from the default feed.

Reply:
done!